.github/workflows/phpunit.yml

```diff
@@ -8,28 +8,27 @@ jobs:
 
     services:
       mysql:
-        image: mysql:5.7.27
+        image: mysql:latest
```

Comment:
Do you have a particular reason to prefer `latest` over a specific version? I think it's better if we pin the version and update as needed. There was an issue just a couple days ago precisely because of that in the main frontity repo: https://github.com/frontity/frontity/pull/750

Reply:
OK, I'll change that. :+1:

Reply:
BTW, I had to change that version because of some [weird error](https://serverfault.com/questions/1016796/cannot-conect-mysql-error-2026-after-upgrade-to-ubuntu-20-04) related to Ubuntu 20.04. Turns out that we were using [`ubuntu-latest`](https://github.com/frontity/wp-plugins/blob/cd16229f521251abd3991eac20115f280d7db418/.github/workflows/phpunit.yml#L7) as well, haha. I'll specify that version too.

Reply:
thanks @DAreRodz :)